Orange/widgets/data/owcorrelations.py

```diff
@@ -67,22 +72,39 @@ def get_clusters_of_attributes(self):
         data = Normalize()(self.data).X.T
         kmeans = KMeans(n_clusters=self.n_clusters, random_state=0).fit(data)
         labels_attrs = sorted([(l, i) for i, l in enumerate(kmeans.labels_)])
-        for _, group in groupby(labels_attrs, key=lambda x: x[0]):
-            group = list(group)
-            if len(group) > 1:
-                yield list(pair[1] for pair in group)
+        return [Cluster(instances=list(pair[1] for pair in group),
+                        centroid=kmeans.cluster_centers_[l])
+                for l, group in groupby(labels_attrs, key=lambda x: x[0])]
```

Comment:
Since the widget imports `attrgetter`, it *could* also import `itemgetter` and use it here.